### PR TITLE
Include procps by default for Visual Studio Code

### PR DESCRIPTION
**Pull request by serl**
# Proposed Changes

Seems like the `ps` command included in Alpine is not enough anymore.
~~Adding this to the documentation, but maybe it makes more sense to have it in the Dockerfile?~~
Adding `procps` to the Dockerfile, so that's included and Visual Studio Code remote-ssh can connect out-of-the-box

## Related Issues

Fixes https://github.com/hassio-addons/addon-ssh/issues/722

### CODE REVIEW

File: ssh/DOCS.md [hunk 1]
```diff
@@ -255,8 +255,11 @@ Setting the following parameters as is shown will allow you to connect to
 your Home Assistant instance using VSCode Remote - SSH:
 
 ```yaml
-allow_remote_port_forwarding: true
-allow_tcp_forwarding: true
+ssh:
+  allow_remote_port_forwarding: true
+  allow_tcp_forwarding: true
+packages:
+  - procps
```
Comment (frenck) [inline ssh/DOCS.md:262]: Honestly, if this is the way to make newer version work, we should adjust the add-on. Not the documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

File: ssh/DOCS.md (hunk 1 above)
Comment (serl) [reply ssh/DOCS.md]: I see. TBH, that wasn't my initial instinct as this addon is "Advanced SSH" and not "Visual Studio Code server in disguise" 😁. So it made sense to me to do the extra step in the configuration.
That said, if `procps` is already installed in the container, I guess it's even easier.

I edited the PR. I got the version to install by `apk info procps`, which actually talks about `procps-ng`, but it seems to work anyways. I'm not that familiar with Alpine, so feel free to correct/ask for corrections as you see fit.
And thanks for everything 🙂

Comment (frenck) [reply ssh/DOCS.md]: Nah this is fine 👍

**Review by frenck — APPROVED** [review]
Thanks, @serl 👍 

../Frenck

**Comment by frenck** [comment]
Why did you push a commit? Was it needed? Something broken? Was there a conflict?

**Comment by serl** [comment]
> Why did you push a commit? Was it needed? Something broken? Was there a conflict?

I saw that it didn't build and that the errors were around packages that were updated on `main`, so I gave the "Update branch" button a shot. I'm sorry if that's wasn't a good idea. Should I remove the extra commit?

**Comment by frenck** [comment]
Thanks for the info, sorry, wasn't sure. As it caused a change after approval, without any comment.

**Review by frenck — APPROVED** [review]

**Pull request merged**